templates/libreswan-host-to-host.conf.j2

```diff
@@ -1,26 +1,27 @@
 #jinja2: lstrip_blocks: True
 {% for tunnel in vpn_connections %}
 {%   if item in tunnel.hosts %}
+{%     set otherhost = item %}
+{%     if tunnel.hosts[item] is mapping and 'hostname' in tunnel.hosts[item] %}
+{%       set otherhost = tunnel.hosts[item].hostname %}
+{%     elif hostvars[item] is defined and hostvars[item].ansible_host is defined %}
+{%      set otherhost = hostvars[item].ansible_host %}
```

Comment:
indentation here

Reply:
Also, you could do this as a 1-liner:
```
 {%     set otherhost =  tunnel.hosts[item] | d(hostvars[item].ansible_host | d(item)) %}
```

Reply:
Neat trick. I restructured some of the other code in the template files using this and it looks much less cluttered now